ephios/event_management/templates/event_management/event_archive.html

```diff
@@ -1,27 +1,25 @@
 {% extends "base.html" %}
+{% load static %}
 {% load i18n %}
 {% load bootstrap4 %}
 
 {% block title %}
     {% trans "Past events" %}
 {% endblock %}
 
-{% block javascript %}
-    {% include "event_management/fragments/datatable.html" with order_column=2 order_direction="desc" %}
-{% endblock %}
-
 {% block content %}
     <div class="page-header">
         <h1>{% trans "Past events" %}</h1>
     </div>
-    <table id="event_table" class="table table-striped display datatable">
+    {% get_current_language as current_language %}
+    <table id="event_table" class="table table-striped display datatable" data-order="[[ 2, &quot;desc&quot; ]]" {% if current_language == "de-de" %} data-language="{&quot;url&quot;: &quot;{% static "datatables/german.json" %}&quot;}" {% endif %} >
```

Comment:
This line looks really hacky and is specific to German. Can you explain what's going on and why it needs to be this way?

Reply:
you need to tell datatables which language to use. Unfortunately they dont't stick to the short codes for languages that django uses. I just moved it here from the old template snippet and thought that it's gonna be okay for the moment as we only have german translations anyway.

Reply:
and we can't do that in our databales initializer in JS?

Reply:
could do, but then we'd need to pass the current_language from django to JS which is a bit ugly with the CSP enabled

Reply:
It's needn't be ugly, I've seen that before. Put a `data-language` tag on the html `body` element and read that from JS. :)